### PR TITLE
Normalize Challenge Properties for COM1000

### PR DESCRIPTION
**Pull request by SaintPeter**
Does two things:
1) Normalizes the order of keys in all challenge .json files
2) Adds default keys to all challenges in all challenge .json files

Tested Locally.

### CODE REVIEW

**Comment by ltegman** [comment]
I thought the 'necessary' keys were: 
```
"id": "",
"title": "",
"description": [],
"challengeSeed": [],
"tests": [],
"type": "",
"challengeType": 0
```
But this PR is still adding things like `solutions` and `MDNlinks` to things like step challenges that don't need it.

**Comment by SaintPeter** [comment]
@benmcmahon100 I can't even remember what is what anymore.

Am I smoking crack?  Is @ltegman correct?

**Comment by benmcmahon100** [comment]
Yeah looks correct. This is the file that handles static properties https://github.com/FreeCodeCamp/COM1000/blob/production/public/headerConfig.json

**Comment by SaintPeter** [comment]
Right, but if those static properties are supposed to be correct, then why are non-static properties being kept?

I ran these through the latest "Production" COM1000.

**Comment by benmcmahon100** [comment]
@SaintPeter are they in your config file? Otherwise I'm not sure what would cause this

**Comment by SaintPeter** [comment]
Sigh.  I didn't update my copy of production somehow.  Fixing.  Leave this in blocked status and I'll fix it.

**Comment by SaintPeter** [comment]
Ok, should be fixed.  QA away.

**Comment by bugron** [comment]
@SaintPeter do we still need the `difficulty` property?

**Comment by SaintPeter** [comment]
No, we do not need it.

File: seed/challenges/03-back-end-development-certification/automated-testing-and-debugging.json [hunk 1]
```diff
@@ -4,46 +4,37 @@
...
       "descriptionEs": [
         "Tanto Chrome como Firefox tienen excelentes consolas JavaScript, también conocidas como DevTools, para depurar tu código JavaScript.",
         "Puedes encontrar las Herramientas para desarrolladores (<code>Developer tools</code>) en el menú de Chrome o la Consola web (<code>Web Console</code>) en el menú de FireFox. Si estás utilizando un navegador diferente, o un dispositivo móvil, nuestra recomendación es que cambies a la versión de escritorio de Firefox o Chrome.",
         "Vamos a imprimir en esta consola utilizando el método <code>console.log</code>.",
         "<code>console.log('Hello world!')</code>"
-        ],
-      "namePt": "",
-      "descriptionPt": []
+      ],
+      "difficulty": "0"
```
Comment (bugron) [inline seed/challenges/03-back-end-development-certification/automated-testing-and-debugging.json]: Number one to be removed.

File: seed/challenges/03-back-end-development-certification/automated-testing-and-debugging.json [hunk 2]
```diff
@@ -80,9 +65,8 @@
         "<code>console.log(typeof(0));</code>",
         "<code>console.log(typeof([]));</code>",
         "<code>console.log(typeof({}));</code>"
-        ],
-      "namePt": "",
-      "descriptionPt": []
+      ],
+      "difficulty": "0"
```
Comment (bugron) [inline seed/challenges/03-back-end-development-certification/automated-testing-and-debugging.json]: Number two to be removed.

**Comment by SaintPeter** [comment]
Removed "difficulty" properties.  There were only 2 left in the whole codebase.

**Comment by SaintPeter** [comment]
I'll close this out until all of the big re-arrangement changes have occurred.

**Comment by bugron** [comment]
@SaintPeter will you work on this (just asking)?

**Comment by SaintPeter** [comment]
Sure - I'd been waiting for the code to stabilize.  I'll get it out this AM.

**Pull request closed without merging**